.github/workflows/ci-linux-static-old-local.yml

```diff
@@ -40,6 +40,11 @@ jobs:
         profile: minimal
         toolchain: stable
         override: true
+    - name: Install dependencies (Linux)
+      if: runner.os == 'Linux'
```

Comment:
Nit: The Linux test can be omitted. See line 48 and line 51.

Reply:
Done